src/core/src/package_managers/PackageManager.py

```diff
@@ -371,11 +374,15 @@ def try_mitigate_issues_if_any(self, command, code, out):
     def check_known_issues_and_attempt_fix(self, output):
         """ Checks if issue falls into known issues and attempts to mitigate """
         return True
-
     # endregion
 
+    @abstractmethod
+    def is_reboot_pending(self):
+        """ Checks if there is a pending reboot on the machine. """
+        pass
```

Comment:
False code coverage check as abstract method is never itself executed. Comment will be closed at review conclusion.